src/Organisms/Modal/ModalBackground.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+import PropTypes from 'prop-types'
+import * as React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: ${(props) => props.theme['modalBackgroundColor']};
+  z-index: 2000;
+  transform: translateZ(0);
```

Comment:
why that `transform: translateZ(0);`?

Reply:
its a css trick to remove the flickering effect of `fixed` elements when you scroll